config/tomlConfig_test.go

```diff
@@ -619,8 +619,8 @@ func TestEnableEpochConfig(t *testing.T) {
     # CreateNFTThroughExecByCallerEnableEpoch represents the epoch when nft creation through execution on destination by caller is enabled
     CreateNFTThroughExecByCallerEnableEpoch = 47
 
-	# CleanUpSCRDataEnableEpoch represents the epoch the informational scrs are cleaned from miniblocks
-	CleanUpSCRDataEnableEpoch = 48
+	# CleanUpInformativeSCRsEnableEpoch represents the epoch when the scrs which contain only information are cleaned from miniblocks and logs are created from it
```

Comment:
`# CleanUpInformativeSCRsEnableEpoch represents the epoch when the informative-only scrs are cleaned from miniblocks and logs are created from them`

Reply:
done